AK/SIMDFormat.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2025, Lucas Chollet <lucas.chollet@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Format.h>
+
+namespace AK {
+
+template<SIMD::SIMDVector V>
+struct Formatter<V> : Formatter<StringView> {
+    ErrorOr<void> format(FormatBuilder& builder, V const& vector)
+    {
+        TRY(builder.put_literal("{ "sv));
```

Comment:
Sorry I missed this, but why is there a space after the `{`?
The test case doesn't expect a space there but still succeeds somehow.

Reply:
> Sorry I missed this, but why is there a space after the `{` ?

I missed it too, don't feel bad :sweat_smile:

> FormatBuilder::put_literal seems to skip characters after braces

Yeah, it almost seems like it was an internal API that was made public.

I removed the space for now.